src/main/java/com/example/tournamentMaker/tournament/round/CupRoundService.java

```diff
@@ -0,0 +1,39 @@
+package com.example.tournamentMaker.tournament.round;
+
+import com.example.tournamentMaker.tournament.Tournament;
+import com.example.tournamentMaker.tournament.game.Game;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class CupRoundService {
+
+    public void createNextRoundSchedule(Tournament tournament, Round currentRound) {
+        int newRoundNumber = currentRound.getTurn();
+        Round newRound = tournament.getRounds().get(newRoundNumber);
+        int gamesAmount = currentRound.getGames().size();
+
+        for (int i = 0; i < gamesAmount; i++) {
+            Game game = currentRound.getGames().get(i);
+            long firstWinner = findWinner(game);
+            i++;
```

Comment:
??

Reply:
iterowanie iteratora w pętli niedopuszczalne? :p
Po prostu z każdych 2 gier w current rundzie robi się jedna w następnej i tak łatwiej mi to było zrobić, ale mogę zamienić np. na while